container/apptabs.go

```diff
@@ -306,9 +308,7 @@ func (r *appTabsRenderer) buildOverflowTabsButton() (overflow *widget.Button) {
 		}
 
 		r.appTabs.popUpMenu = buildPopUpMenu(r.appTabs, overflow, items)
```

Comment:
I'm not certain but it looks like the overflow variable is no longer set. 
Is this part of named returns magic?

Reply:
I'd say that it's mostly an oversight. I don't know why it didn't cause my issues as far as I could see.

Reply:
> Is this part of named returns magic?

Yes, it is, named returns are hard to understand IMHO 😕

> I don't know why it didn't cause my issues as far as I could see.

Because `overflow` is set when the function `returns`. It is certainly not very obvious but it is right. The overflow is set.
https://play.golang.org/p/5tTC9YSiwpm

Reply:
So, should we a) keep it as is, b) add a comment or c) remove the named return revert to how it was prior?

Reply:
I'll go with c) then. I think it is a bit unclear as well :slightly_smiling_face:

Reply:
I had to keep the named return, but removed the magic part about it. Should be good now.

Reply:
Optionally, you could remove the named return too and just have `buildOverflowTabsButton() *widget.Button`

Reply:
Nope. Can't access the variable inside the anonymous function by doing it that way (without modifying it in more ways). I think this works good as is now.